### ISSUE TITLE
Nixtoons2

### ISSUE THREAD

**Issue by Rhysowen83** [doko-desuka/plugin.video.watchnixtoons2#15]
Hello! What an amazing addon and works perfectly thank you! 

I have one annoying and recent gripe though, why is the newest Attack on Titan episodes (dubbed -season 3 episodes 15 onwards) not showing up to watch? It’s 2 weeks behind now and I know they’re available and been aired, I can’t find anything online about it and no other way to contact the devs to enquire. 

This is the english dubbed version just to be clear, the subbed version is updating fine with no issues. Thanks in advance 👍🏻

**Comment by doko-desuka** [comment]
Hi @Rhysowen83,
this add-on scrapes content from the WatchCartoonOnline website, and I'm not related to them at all (though I do point people to their donation page, to keep them going).  

If they didn't put up the episodes yet there must be some reason. Attack on Titan seems to have S3E16, is that some progress?
Anyway, you can reach them for support on this email: **admin at watchcartoononline dot io**
(But don't say you're coming in from this add-on, just act like you're a normal website visitor.)

**Comment by Rhysowen83** [comment]
Thank you so much for your help and my sincere apologies for the late reply, I got your email and thought I had replied but hadn’t! 

Yes shortly after messaging you the episodes updated and to 16 which was up to date and I’m now just waiting for the next one. It must have been a stroke of luck they updated after I got in touch ha! 

Thanks again you’re a star! If I need anymore support from them I’ll use the contact you provided, keep up the good work, great repo and thoroughly enjoy the addon :-) 

Rhys
